src/messaging/ReliableMessageMgr.cpp

```diff
@@ -471,6 +465,27 @@ CHIP_ERROR ReliableMessageMgr::MapSendError(CHIP_ERROR error, uint16_t exchangeI
     return error;
 }
 
+void ReliableMessageMgr::CalculateNextRestranTime(RetransTableEntry * entry)
```

Comment:
Suggest `CalculateNextRetransTime` to account for the fact that `entry->nextRetransTime` is what is touched.

Reply:
done